mappings/net/minecraft/registry/HolderProvider.mapping

```diff
@@ -31,3 +33,9 @@ CLASS net/minecraft/unmapped/C_pzdchrcy net/minecraft/registry/HolderProvider
 			COMMENT {@return the holder provider of the given registry, may be empty if the registry does not exist in this provider}
 			ARG 1 registryKey
 				COMMENT the key of the registry
+		METHOD m_yuymjcux get (Lnet/minecraft/unmapped/C_xhhleach;Lnet/minecraft/unmapped/C_xhhleach;)Ljava/util/Optional;
+			COMMENT {@return the entry's holder by its key, may be empty if the entry does not exist in the searched registry}
+			ARG 1 registry
+				COMMENT the key of the registry to search in
+			ARG 2 key
+				COMMENT the key of the entry to find
```

Comment:
```suggestion
			ARG 1 registryKey
			ARG 2 entryKey
```
It seems like the comments were only necessary because the names weren't quite descriptive enough.

Reply:
I also saw that everything else in the class had docs, so I wanted consistency

Reply:
Hm, it seems both of my suggestions are inconsistent with the rest of the class.